src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -38,6 +39,7 @@
 	public RecheckSeleniumAdapter() {
 		System.setProperty( "de.retest.ignoredAttributes", "outline" );
 		defaultValuesProvider = new DefaultValuesProvider();
+		logger.debug( "New RecheckSeleniumAdapter created: " + System.identityHashCode( this ) );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2629))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=77&issues=AWaHMZUUlB6-zLfzofF7&open=AWaHMZUUlB6-zLfzofF7)

[ISSUE_KEY]: AWaHMZUUlB6-zLfzofF7